glean-core/src/upload/mod.rs

```diff
@@ -219,6 +225,16 @@ impl PingUploadManager {
         self.processed_pending_pings.load(Ordering::SeqCst)
     }
 
+    /// Gets the ping queue and checks in a ping with a certain document_id is already enqueued.
+    fn has_enqueued(
```

Comment:
I think `is_enqueued` is a better name.

Reply:
or maybe `queue_contains`. 
Nah, I think I like the first better.

Reply:
Hum, funny. `is_enqueued` is the name I initally gave to this function and then changed it.

Anyways, I have no strong opinions here, I'll change it.